module/calcDistancePoints.test.js

```diff
@@ -0,0 +1,36 @@
+const assert = require("assert");
+const calcDistancePoints = require("./calcDistancePoints");
+
+describe("calcDistancePoints", () => {
```

Comment:
it should looks something like this: "when we want calc distance point with calcDistancePoints function"